### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by saiutkarsh33**
- Added a few lines to the DG with respect to set course

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -204,6 +204,9 @@ Step 4. The user now decides that adding the person was a mistake, and decides t
 
 <puml src="diagrams/UndoRedoState3.puml" alt="UndoRedoState3" />
 
+Step 5. The user wants to set the course name. He enters the command setcrs CS2103T, causing the course name to appear on the main window's title.
```
Comment (ForAeons) [inline docs/DeveloperGuide.md:207]:
```suggestion
Step 5. The user wants to set the course name. He enters the command `setcrs CS2103T`, causing the course name to appear on the main window's title.
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -477,3 +480,13 @@ students.
...
 
 1. _{ more test cases …​ }_
+
+### Setting the course
+
+1. Setting a course 
+    
+   1. Test case: `setcrs CS2103`<br>
+      Enter setcrs followed by a whitespace, followed by a course code in the format "XX1234Y", where X and Y can be any letter
+      in upper or lower case, and 1234 can be any 4 digit number. Y is optional
```
Comment (ForAeons) [inline docs/DeveloperGuide.md:490]:
```suggestion
      Enter setcrs followed by a whitespace, followed by a course code in the format `XXYYYYZ`, where X and Z can be any letter
      in upper or lower case, Z is optional and YYYY can be any 4 digit number.
```

**Review by ForAeons — CHANGES_REQUESTED** [review]
LGTM, just some small nits

**Review by tituschewxj — APPROVED** [review]
Apart from the nits raised by @ForAeons, LGTM!

**Review by ForAeons — APPROVED** [review]

**Pull request merged**